src/blockchain/log-processors/order-filled/calculate-number-of-shares-traded.ts

```diff
@@ -0,0 +1,5 @@
+import BigNumber from "bignumber.js";
+
+export function calculateNumberOfSharesTraded(numShares: string, numTokens: string, price: string): string {
```

Comment:
`Int256`? I know it's the same thing, but it seems like it's better documentation